src/main/java/edu/ucsb/cs56/ucsb_courses_search/service/GithubOrgMembershipService.java

```diff
@@ -30,6 +30,7 @@
     @Autowired
```

Comment:
This entire file looks unnecessary if we're removing GitHub OAuth. We should probably remove it and turn this into a Google OAuth membership service instead. 

We'll probably need to make changes to the MembershipService interface as well... it looks like all of its member functions are specific to GitHub (so I'm questioning why it was listed here and not in GitHubOrgMembershipService instead).

Reply:
We discussed this, but thought that it would be better to do once the Google OAuth was implemented completely, unless you think it is fine to just do now.

Reply:
Makes sense, since OAuth is a blocker for some teams, and we don't have much time, it's better to just get an MVP out sooner and to fully build it later.